### PR TITLE
Improve workflow templates for Connection Pools and correct Email and Web Notification task names

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml [hunk 1]
```diff
@@ -4,10 +4,23 @@
   xmlns="urn:proactive:jobdescriptor:3.11" xsi:schemaLocation="urn:proactive:jobdescriptor:3.11 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.11/schedulerjob.xsd"  name="SQL_Pooled_Connection_Update" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
   <variables>
     <variable name="RDBMS_NAME" value="HSQLDB" model="PA:LIST(PostgreSQL, MySQL, Greenplum, SQL Server, Oracle, HSQLDB)"/>
+    <variable name="HOST" value="localhost" />
+    <variable name="PORT" value="" />
+    <variable name="USERNAME" value="scheduler" />
```
Comment (amouhoub) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml]: maybe change the default value of USERNAME with empty values as they will definitely be changed by the user (in contrast with localhost for HOST which will most probably be used as is)

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml [hunk 2]
```diff
@@ -4,10 +4,23 @@
...
+    <variable name="DATABASE" value="scheduler" />
```
Comment (amouhoub) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml]: Same as above

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml (hunk 1 above)
Comment (aminelouati) [reply DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml]: you are right. However, for SQL_Pooled_Connection_Query.xml do you agree to keep the default values to make the template executable ?

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml [hunk 3]
```diff
@@ -116,132 +124,140 @@ if (!password) {
...
+    dBConnectionPoolsHolder = DBConnectionPoolsHolder.getInstance();
+    rs = dBConnectionPoolsHolder.executeQuery(dbConnectionDetails, sqlStatements)
+} catch (Exception e) {
+    e.printStackTrace();
+    throw e;
+}
+interceptor.stop()
 
+OUTPUT_TYPE = variables.get("OUTPUT_TYPE")
+OUTPUT_FILE = variables.get("OUTPUT_FILE")
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml]: this variable is not used.

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml [hunk 4]
```diff
@@ -116,132 +124,140 @@ if (!password) {
...
 sqlStatements = variables.get("SQL_STATEMENTS")   
 if (!sqlStatements){
     throw new IllegalArgumentException("ERROR: SQL_STATEMENTS variable is not provided by the user. Empty value is not allowed.")
 }
+ResultSet rs = null
+try {
+    dBConnectionPoolsHolder = DBConnectionPoolsHolder.getInstance();
+    rs = dBConnectionPoolsHolder.executeQuery(dbConnectionDetails, sqlStatements)
+} catch (Exception e) {
+    e.printStackTrace();
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml]: I am afraid this printStackTrace works only in fork mode.

in non-fork mode:
println "something" works
System.out.println("something") does not work, and printStackTrace() I believe is the same.

Could you check ?

Comment (aminelouati) [reply DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml]: yes in non fork mode printStackTrace() does not work

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml (hunk 1 above)
Comment (amouhoub) [reply DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml]: @aminelouati yeah sure, I was just talking about `USERNAME` and `DATABASE`

**Pull request merged**